app/assets/hee/blocks/content/main/nhsuk-hee-blocklinks/_blocklinks.scss

```diff
@@ -166,4 +166,12 @@
 			}
 		}
 	}
-}
\ No newline at end of file
+}
+
+.nhsuk-tabs__panel {
```

Comment:
As discussed, could we move this under `app/assets/hee/blocks/content/main/hee-tabs/_tabs.scss` ?